src/sage/rings/polynomial/multi_polynomial_libsingular.pyx

```diff
@@ -340,11 +340,11 @@ cdef class MPolynomialRing_libsingular(MPolynomialRing_base):
             Multivariate Polynomial Ring in x, y, z over
              Ring of integers modulo 25213521351515232
             sage: type(P)
-            <class 'sage.rings.polynomial.multi_polynomial_ring.MPolynomialRing_polydict_with_category'>
+            <class 'sage.rings.polynomial.multi_polynomial_libsingular.MPolynomialRing_libsingular'>
 
             sage: P.<x,y,z> = PolynomialRing(Integers(2^32), order='lex')
             sage: P(2^32-1)
-            4294967295
+            -1
```

Comment:
This is harmless, but technically modifies the behavior. (New behavior is probably better though.)